packages/hash/frontend/src/components/layout/PageHeader/ActionsDropdown.tsx

```diff
@@ -66,6 +66,11 @@ export const ActionsDropdown: React.FC<{
           backgroundColor: open
             ? theme.palette.blue["70"]
             : theme.palette.gray[20],
+
+          "&:hover": {
+            backgroundColor: theme.palette.blue["70"],
+            color: theme.palette.common.white,
+          },
```

Comment:
@aj had to set this here because I noticed we are using IconButtons with different hover/focus styles. As a result placing default styles in MuiIconButtonThemeOptions caused some issues. This is a temporary fix, but should be properly addressed in this task [internal link](https://app.asana.com/0/1201864704093982/1201916223398333)